src/Promitor.Core.Scraping/LogAnalyticsScraper.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Azure.Management.Monitor.Fluent.Models;
+using Microsoft.Extensions.Logging;
+using Promitor.Core.Contracts;
+using Promitor.Core.Contracts.ResourceTypes;
+using Promitor.Core.Metrics;
+using Promitor.Core.Scraping.Configuration.Model.Metrics;
+using Promitor.Integrations.LogAnalytics;
+
+namespace Promitor.Core.Scraping
+{
+    public class LogAnalyticsScraper : Scraper<LogAnalyticsResourceDefinition>
+    {
+        private readonly LogAnalyticsClient _logAnalyticsClient;
+
+        public LogAnalyticsScraper(ScraperConfiguration scraperConfiguration) : base(scraperConfiguration)
+        {
+            _logAnalyticsClient = scraperConfiguration.LogAnalyticsClient;
+        }
+
+        protected override async Task<ScrapeResult> ScrapeResourceAsync(string subscriptionId, ScrapeDefinition<IAzureResourceDefinition> scrapeDefinition, LogAnalyticsResourceDefinition resourceDefinition, AggregationType aggregationType, TimeSpan aggregationInterval)
+        {
+            Guard.NotNull(scrapeDefinition, nameof(scrapeDefinition));
+            Guard.NotNull(scrapeDefinition.LogAnalyticsConfiguration, nameof(scrapeDefinition.LogAnalyticsConfiguration));
+
+            string resourceUri = "logAnalytics";
+            var query = scrapeDefinition.LogAnalyticsConfiguration.Query;
+            var workspaceId = resourceDefinition.WorkspaceId;
+
+            List<MeasuredMetric> measuredMetrics = new List<MeasuredMetric>();
+            try
+            {
+                // Query Azure Monitor for metrics
+                var result = await _logAnalyticsClient.QueryDouble(workspaceId, query, aggregationInterval);
+                var measuredMetric = MeasuredMetric.CreateWithoutDimension(result);
+                measuredMetrics.Add(measuredMetric);
+            }
+            catch (Exception e)
+            {
+                Logger.LogError(e.Message);
+                Logger.LogWarning("Something wrong when query logs analytics");
```

Comment:
I'd do either, preferrably this:
```suggestion
                Logger.LogError(e);
```